controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -78,6 +78,9 @@ type DataScienceClusterConfig struct {
 
 const (
 	finalizerName = "datasciencecluster.opendatahub.io/finalizer"
+	RHOAIAppNS    = "redhat-ods-applications"
+	ODHAppNS      = "opendatahub"
+	MonitoringNS  = "redhat-ods-monitoring"
```

Comment:
Those changes do not seem to be related with the PR :)

Reply:
yeah, the lint is annoying, to get rid of the `//nolint:goconst`
but i can revert it

Reply:
reverted